Bottom: 04ed8fed5e91c4ff26ea0b45d381e559d2816034
Top:    4e1403556b5bf1d95db1a270125b99740ed17edf
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-17 16:39:48 -0500

Refresh of TESTING--VirtualMachine

---
diff --git a/Vagrantfile b/Vagrantfile
index 78f9816..af48105 100644
--- a/Vagrantfile
+++ b/Vagrantfile
@@ -6,6 +6,8 @@ Vagrant.configure("2") do |config|
         main.vm.hostname = "vmdaedalus.novatech-llc.com"
         main.vm.provider "virtualbox" do |vb|
             vb.name = "vmdaedalus"
+            vb.memory = 4096
+            vb.cpus = 4
         end
     #    config.vm.network "public_network", ip: "192.168.0.100", netmask:"255.255.0.0", bridge: "enp0s31f6"
         main.vm.network "private_network", ip: "192.168.0.100", netmask:"255.255.0.0"
@@ -26,28 +28,4 @@ Vagrant.configure("2") do |config|
 
     end
 
-    config.vm.provision "ansible" do |ansible|
-        ansible.verbose = "v"
-        ansible.playbook = "ansible-playbook/site.yml"
-        ansible.groups = {
-            "bacula" => ["vmdaedalus.novatech-llc.com"],
-            "docker-hosts" => ["vmdaedalus.novatech-llc.com"],
-            "buildbot" => [
-                "vmdaedalus.novatech-llc.com",
-                "buildbot-worker.novatech-llc.com"
-            ],
-            "docker-hosts:vars" => {
-                "openssl_image_repo" => "https://github.com/AndrewCooper/docker-openssl.git",
-                "openldap_image_repo" => "https://github.com/AndrewCooper/docker-openldap.git"
-            }
-        }
-        # ansible.tags = []
-        # ansible.skip_tags = ["restore"]
-        ansible.host_vars = {
-            "vmdaedalus.novatech-llc.com" => {
-                "network_connection" => "System enp0s8"
-            }
-        }
-    end
-    
 end
